src/common/install.rs

```diff
@@ -39,3 +42,44 @@ pub fn populate_data_dir(data_dir: &Path) {
         }
     }
 }
+
+pub fn default_config_dir() -> Option<PathBuf> {
+    vec![
```

Comment:
Could maybe use an `or_else()` chain here if you'd like. Similar to [here](https://github.com/zellij-org/zellij/blob/169e25af66fe534a5091583b4b5f62dd1b7eea9b/src/common/mod.rs#L473)

Also, does just an array work here? Just `[]` instead of `vec![]`? I suppose you wouldn't need either with the `or_else()` chain though :slightly_smiling_face:

Reply:
Cheers, that is good! I'll try that out.

Reply:
I don't see a way to implement that or_else() in a good way here. I want the contents, in this case the path, not the value that is returned.